src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_help.py

```diff
@@ -282,6 +449,8 @@
             For resource group:         --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1
             For EventGrid topic:        --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.EventGrid/topics/t1
             For storage account:        --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.Storage/storageaccounts/sa1
+            For EventGrid domain:       --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.EventGrid/domains/d1/topics/t1
```

Comment:
Minor typo: Remove /topics/t1 suffix at the end.

Reply:
Fixed